agent/config/pbench-agent.cfg.example

```diff
@@ -40,8 +40,16 @@ interval = 30
 # pandas-package = python-pandas
 # Fedora has python2-pandas and python3-pandas
 # pandas-package = python2-pandas
+stockpile_dependencies = git, ansible
 
 [config]
 path = %(pbench_install_dir)s/config
 files = pbench-agent.cfg
 
+[stockpile]
+stockpile_user = root 
+local_remote_user = root 
+host_remote_user = root
+stockpile_path = /var/lib/pbench-agent
+stockpile_log =
```

Comment:
Can you add a comment that `stockpile_log` and `stockpile_output_path` are optional, the default being ... (whatever the default is)?

Reply:
Done.